bazel_integration_test/script_test.bzl

```diff
@@ -0,0 +1,91 @@
+load(
+    "//bazel_integration_test:defs.bzl",
+    "bazel_integration_test",
+)
+
+_RUNNER_SCRIPT = """
+#!/bin/bash
+
+script="%s"
+bazel="${BIT_BAZEL_BINARY:-}"
+
+[[ -n "${bazel:-}" ]] || exit_with_msg "Must specify the location of the Bazel binary."
+
+working_dir="."
+cd "${working_dir}"
+
+export bazel
```

Comment:
I presume that the `export bazel` allows the script to find the Bazel binary. Is there any reason not to use the `BIT_BAZEL_BINARY` variable that the framework provides? 

As some background, I tried to select an env var name that would not likely be overridden or confused. I worry that the env variable `bazel` might be generic. What do you think?

Reply:
I was trying to decide if `${bazel} info release` or `${BIT_BAZEL_BINARY} info release` was easier to read. I lean towards the former but I can always define the name in my own tests if it seems needed.

Reply:
I don't feel strongly about the name. I used the other name for clarity about how the value was set. Let's keep your version for now.